Assets/HoloToolkit-Examples/MotionControllers-GrabMechanics/Scripts/ScalableTwoHand.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ScalableTwoHand : MonoBehaviour {
```

Comment:
Why is there an empty class?

Reply:
I just realized my comment above applies to something different than what you're referring to. That's my mistake. So in my last comment, I was talking about the two-hand rotatable object that relies almost exclusively on Unity joints (an example that we've since disabled in the scene because it's very finicky). I see now that you're referring to ScalableTwoHand.cs, which is a script that has been replaced by the combination of GrabbableMultiJoint.cs and ScalableObject.cs. I've deleted ScalableTwoHand.cs.